pkg/cheatsheet/generate.go

```diff
@@ -131,7 +131,7 @@ func getBindingSections(bindings []*types.Binding, tr *i18n.TranslationSet) []*b
 			return false
 		}
 
-		return (binding.Description != "" || binding.Alternative != "")
+		return (binding.Description != "" || binding.Alternative != "") && binding.Key != nil
```

Comment:
if a keybinding has no key assigned, we don't want to include it in the cheatsheet